subsys/spm/secure_services.c

```diff
@@ -57,6 +57,8 @@ int spm_secure_services_init(void)
 #define FICR_PUBLIC_SIZE        0xA1C
 #define FICR_RESTRICTED_ADDR    (FICR_BASE + 0x130)
 #define FICR_RESTRICTED_SIZE    0x8
+#define UICR_ADDR		(NRF_UICR_S_BASE)
```

Comment:
Can we align here? We have FICR_BASE, but then NRF_UICR_S_BASE - how could these look similar?

Reply:
I agree with you. FICR_BASE is a local define, a couple of lines up. Would it make sense to use NRF_FICR_S_BASE directly instead?

Reply:
I believe so, yes. Or even, simply NRF_FICR and NRF_UICR, if possible.
 We don't need to overdo it with macro definitions.
@anangl ^^ please, comment

Reply:
`(u32_t)NRF_FICR` and `(u32_t)NRF_UICR` should work here.